modules/update-mirror-repository.adoc

```diff
@@ -9,7 +9,7 @@ Before you upgrade a cluster on infrastructure that you provision in a restricte
 
 .Procedure
 
-. Review the link:https://access.redhat.com/solutions/4583231[{product-title} upgrade paths] to confirm that there is an upgrade path between your current and intended cluster versions.
+. Review the link:https://access.redhat.com/solutions/4583231[{product-title} upgrade paths] to confirm that there is an upgrade path between your current and intended cluster versions. You can use the link:https://access.redhat.com/labs/ocpupgradegraph/update_channel[OpenShift Upgrade Graph visualizer and update planner] to view channel graphs or plan an upgrade from one version to another.
```

Comment:
This kcs already points to this tool, so its a bit redundant to list both of these. 
I would remove the KCS and just point to the new tool.

Reply:
Updated :+1: